browser/chrome/browser/migration/migration.dtd

```diff
@@ -43,4 +44,4 @@
 <!ENTITY done.title                     "Importowanie zakończone">
 <!ENTITY done.label                     "Poniższe elementy zostały zaimportowane:">
 
-<!ENTITY closeSourceBrowser.label "Przed przejściem dalej, oroszę upewnić się, że wybrana przeglądarka jest wyłączona.">
+<!ENTITY closeSourceBrowser.label "Przed przejściem dalej proszę upewnić się, że wybrana przeglądarka jest wyłączona.">
```

Comment:
Still, changing the commit message for an already pushed commit involves force-pushing, which I'd like to avoid.